### PR TITLE
支持Aut

### PR DESCRIPTION
**Pull request by lowrt**

### CODE REVIEW

File: WzComparerR2.MapRender/UI/Tooltip2.cs [hunk 1]
```diff
@@ -393,6 +395,11 @@ private TooltipContent DrawItem(GameTime gameTime, RenderEnv env, UIWorldMap.Too
                         var icon = Content.Load<Texture2D>(nameof(MRes.UIWindow_img_ToolTip_WorldMap_ArcaneForce));
                         addBarrier(spotBarrierArc, icon, new Color(221, 170, 255));
                     }
+                    else if (spotBarrierAut > 0)
+                    {
+                        var icon = Content.Load<Texture2D>(nameof(MRes.UIWindow_img_ToolTip_WorldMap_AuthenticForce));
+                        addBarrier(spotBarrierAut, icon, new Color(221, 170, 255));
```
Comment (Kagamia) [inline WzComparerR2.MapRender/UI/Tooltip2.cs:401]: Is there any screenshot for this forecolor?

**Comment by Kagamia** [comment]
just merged.

since I've not update to KMST1112 and can't verify these changes 🤣

File: WzComparerR2.MapRender/UI/Tooltip2.cs (hunk 1 above)
Comment (lowrt) [reply WzComparerR2.MapRender/UI/Tooltip2.cs:401]: 螢幕擷取畫面 2020-12-15 093238

🤣

**Comment by KENNYSOFT** [comment]
44344b024fa7070fed09649a23227ea9bedf6756 c43290d1bdae86a48eafc22d4547ff71e0a04397

History mixed up for GearPropType.cs and ItemStringHelper.cs 😭

**Comment by Kagamia** [comment]
@KENNYSOFT  I've tried to squash the commits locally but only find 1 line inserted.

So I've no idea why this pr detects so much changes.

**Comment by KENNYSOFT** [comment]
I found it: line ending changed.

image

It changed from LF to CRLF.

**Comment by Kagamia** [comment]
🤣

**Comment by KENNYSOFT** [comment]
Since .gitattributes file states `text=auto`, I think it should be replaced with LF, as before.
Can you make the changes?

**Comment by lowrt** [comment]
I may upload that file directly🤣

**Pull request merged**